utils.py

```diff
@@ -37,51 +37,75 @@ def connect_to_mysql(host, user, password):
     except Exception as e:
         print("Error: ", e)
 
-#function that creats database
-        
-def create_database(cursor, database_name):
-    # drop database if exists
+# Define a function that creates a new MySQL database.        
+def create_database(cursor: mysql.cursor.MySQLCursor, database_name: str) -> None:
+    """
+    Creates a new MySQL database with the given name.
+
+    Args:
+        cursor (mysql.cursor.MySQLCursor): The cursor object to use to execute SQL statements.
+        database_name (str): The name of the new database to create.
+
+    Returns:
+        None
+    """
+    # Drop the database if it already exists
     drop_db_query = f'DROP DATABASE IF EXISTS {database_name}'
     cursor.execute(drop_db_query)
 
-    # create database
+    # Create the database
     create_db_query = f'CREATE DATABASE {database_name}'
     cursor.execute(create_db_query)
 
-    # show databases
+    # Show all databases on the MySQL server
     cursor.execute('SHOW DATABASES')
     databases = cursor.fetchall()
     print("List of databases:")
     for db in databases:
         print(db[0])
```

Comment:
You don't need to loop through if you want to display all the databases that exist. Remove for loop and in the print just do  `print("List of databases:", databases)`